crates/swc_ecma_minifier/tests/fixture/next/47005/input.js

```diff
@@ -0,0 +1,539 @@
+                            for (; ot(g) && !le(g);) {
+                                let w = G(g),
+                                    T = re(g);
+                                w.position === "fixed" ? p = null : (y ? T || p : T || w.position !== "static" || !p || !["absolute", "fixed"].includes(p.position)) ? p = w : d = d.filter(x => x !== g), g = kt(g)
+                            }
+                            return a.set(u, d), d
+                        }(e, this._c) : [].concat(n), i = [...r, o], s = i[0], f = i.reduce(() => {
+                        }, _e(e, s, l));
+                        return {
+                            width: f.right - f.left,
+                            height: f.bottom - f.top,
+                            x: f.left,
+                            y: f.top
+                        }
+                    },
+                    convertOffsetParentRelativeRectToViewportRelativeRect: function () {
+
+                    },
+                    isElement: ot,
+                    getDimensions: function (t) {
+                        return Y(t) ? Ce(t) : t.getBoundingClientRect()
+                    },
+                    getOffsetParent: Pe,
+                    getDocumentElement: nt,
+                    getScale: Tt,
+                    async getElementRects(t) {
+                        let {
+                            reference: e,
+                            floating: n,
+                            strategy: o
+                        } = t, l = this.getOffsetParent || Pe, r = this.getDimensions;
+                        return {
+                            reference: To(e, await l(n), o),
+                            floating: {
+                                x: 0,
+                                y: 0,
+                                ...await r(n)
+                            }
+                        }
+                    },
+                    getClientRects: t => Array.from(t.getClientRects()),
+                    isRTL: t => G(t).direction === "rtl"
+                };
+                var se = (t, e, n) => {
+                    let o = new Map,
+                        l = {
+                            platform: xo,
+                            ...n
+                        },
+                        r = {
+                            ...l.platform,
+                            _c: o
+                        };
+                    return Ee(t, e, {
+                        ...l,
+                        platform: r
+                    })
+                };
+                var ce = async ({
+                    elementReference: t = null,
+                    tooltipReference: e = null,
+                    tooltipArrowReference: n = null,
+                    place: o = "top",
+                    offset: l = 10,
+                    strategy: r = "absolute",
+                    middlewares: i = [oe(Number(l)), ee(), ne({
```

Comment:
The problem was that, `oe` was inlined into this arrow head and the variables created from the injector is appended to the arrow body.